tests/onnxruntime/test_stable_diffusion_pipeline.py

```diff
@@ -483,3 +488,55 @@ def test_vae_image_processor_pil(self):
                 in_np = np.array(i)
                 out_np = to_np(out) if output_type == "pil" else (to_np(out) * 255).round()
                 self.assertTrue(np.allclose(in_np, out_np, atol=1e-6))
+
+
+@unittest.skipIf(parse(_diffusers_version) <= Version("0.21.4"), "not supported with this diffusers version")
+class ORTLatentConsistencyModelPipelineTest(ORTModelTestMixin):
+    SUPPORTED_ARCHITECTURES = [
+        "latent-consistency",
+    ]
+    ORTMODEL_CLASS = ORTLatentConsistencyModelPipeline
+    TASK = "text-to-image"
+
+    @parameterized.expand(SUPPORTED_ARCHITECTURES)
+    @require_diffusers
+    def test_compare_to_diffusers(self, model_arch: str):
+        ort_pipeline = self.ORTMODEL_CLASS.from_pretrained(MODEL_NAMES[model_arch], export=True)
+        self.assertIsInstance(ort_pipeline.text_encoder, ORTModelTextEncoder)
+        self.assertIsInstance(ort_pipeline.vae_decoder, ORTModelVaeDecoder)
+        self.assertIsInstance(ort_pipeline.vae_encoder, ORTModelVaeEncoder)
+        self.assertIsInstance(ort_pipeline.unet, ORTModelUnet)
+        self.assertIsInstance(ort_pipeline.config, Dict)
+
+        from diffusers import LatentConsistencyModelPipeline
```

Comment:
This is taste, but maybe imports at the top with a `if_diffusers_is_available()` (or maybe not even needed given that the test is all about diffusers) is preferable.

https://peps.python.org/pep-0008/#imports

Reply:
I see. Then what I would do is a try/except import, and add a decorator (or condition + skiptest) on latent consistency tests.

Reply:
we already use the skip decorator, concerning your try/except import suggestion I'm not sure to get why it would be cleaner in this case, could you elaborate ? (I'd rather add a diffusers version check or keep as is)

Reply:
It would be cleaner because it would respect PIP 8.

Reply:
Oh, I was referring to something like:

```python
try:
    from diffusers import LatentConsistencyModelPipeline
except Exception:
    LatentConsistencyModelPipeline = None
```